config.sample.yaml

```diff
@@ -30,6 +30,12 @@ ircService:
       ssl: true
       # Whether or not IRC server is using a self-signed cert or not providing CA Chain
       sslselfsign: false
+      # A specific CA to trust instead of the default CAs. Optional.
+      ca: |
+        -----BEGIN CERTIFICATE-----
+        ...
+        -----END CERTIFICATE-----
```

Comment:
Can you wrap these in `#` comments please? Otherwise, if people blindly `cp config.sample.yaml config.yaml` they will use this as their `ca`!